src/schemas/user.schema.js

```diff
@@ -2,15 +2,57 @@ var mongoose = require('mongoose');
 var Schema = mongoose.Schema;
 
 var userSchema = new Schema({
-  name: {
+  Id: {
     type: String,
     required: true
   },
+  Fname: {
+    type: String,
+    required: true
+  },
+  Mname: {
+    type: String,
+    required: false
+  },
+  Lname: {
+    type: String,
+    required: true
+  },
+  Bday: {
+    type: String,
```

Comment:
this should be a type `Date`
you can also add `default` property for birthDate
so this will be
`{type: Date, default: Date.now, required: true}`

Reply:
default: Date.now  - this means every insert of document, this will be automatically add a date now

Reply:
I don't think date now is good to be a default of the user's birthday. Unless you are just giving an example. But I get what you are triying say.

Reply:
oh yes you're right, we will remove the default property

Reply:
I think it should be required. Because you are applying for a job. So this kind of data is important to the employer and it is a kind of data that every employee have.

Reply:
yes you are right we should probably had this a required